src/xml_file_loader.cc

```diff
@@ -326,6 +327,28 @@ void XMLFileLoader::LoadRecipes() {
   }
 }
 
+void XMLFileLoader::LoadPackages() {
+  InfileTree xqe(*parser_);
+  // create default package
+  ctx_->AddPackage("default", 0, std::numeric_limits<int>::max(), "first");
+
+  std::string query = "/*/package";
+  int num_packages = xqe.NMatches(query);
+  for (int i = 0; i < num_packages; i++) {
+    InfileTree* qe = xqe.SubTree(query, i);
+    std::string name = cyclus::OptionalQuery<std::string>(qe, "name", "default");
+    CLOG(LEV_DEBUG3) << "loading package: " << name;
+    
+    double fill_min = cyclus::OptionalQuery<double>(qe, "fill_min", 0);
+    double fill_max = cyclus::OptionalQuery<double>(qe, "fill_max", std::numeric_limits<double>::max());
+    
+    std::string strategy = cyclus::OptionalQuery<std::string>(qe, "strategy", "first");
+
+    boost::shared_ptr<Package> p = Package::Create(name, fill_min, fill_max, strategy);
```

Comment:
It might make sense to put this in a `ReadPackage()` method analogous to `ReadRecipe()` but this one is a lot simpler, so could go either way.

Reply:
Yeah my thought was the simplicity negated the need for an extra method at this point